shared/oae/macros/header.html

```diff
@@ -0,0 +1,19 @@
+<!--
+ * Rendering a configurable header with optional controls
+-->
+{macro header(title, action)}
+    <div class="oae-header">
+        <div class="row">
+            <div class="oae-header-title col-xs-12 col-sm-10">
+                <h2>${title}</h2>
+            </div>
+            {if action}
+                <div class="oae-header-action hidden-xs col-sm-2">
+                    <button class="btn btn-link ${action.trigger}" ${%_.map(action.data, function(value, key) { return 'data-' + key + '="' + value + '"'; }).join(' ')%}>
+                        <i class="fa ${action.icon}"></i>
```

Comment:
These actions need descriptive text for accessibility (both a `title` and a `<span class="sr-only"></span>` within the `i`)